app/src/main/java/com/it3048/mobiledeviceproject/services/MeetingService.kt

```diff
@@ -61,9 +60,9 @@ class MeetingService(application: Application) {
 
     }
 
-    internal fun getLocalMeetingDAO() : MeetingDAO {
-        val db = Room.databaseBuilder(application, MeetingDatabase::class.java, "MobileMeetings").build()
-        val localMeetingDAO = db.localMeetingDAO()
-        return localMeetingDAO
+    private fun getLocalMeetingDAO(): MeetingDAO {
+        val db =
+            Room.databaseBuilder(application, MeetingDatabase::class.java, "MobileMeetings").build()
+        return db.localMeetingDAO()
```

Comment:
Cuts out extraneous val defining, when it's only called in the return statement.